src/types/entities/schelling.ts

```diff
@@ -37,72 +37,142 @@ export class SchellingGame {
 	// All players have a unique overlay address
 	private players: BTree_<string, Player>
 	private rounds: BTree_<number, Round>
+	private currentRoundNo = 0
+	private lastBlock: BlockDetails
 
 	private myOverlays: string[] = []
 	private myAccounts: string[] = []
 
+	private static _showLogs = false
+
 	/**
 	 * Make the constructor private so that it can't be called.
 	 * @private
 	 */
 	private constructor() {
 		this.players = new BTree()
 		this.rounds = new BTree()
+		this.currentRoundNo = 0 // We haven't started the game yet
```

Comment:
I wondered if this should start at nil just in case someone ever runs a swarm on a brand new blockchain and actually wants to start at round 0?